src/lib/viewers/doc/_docBase.scss

```diff
@@ -79,6 +79,12 @@
         }
     }
 
+    // Allow other text to be selectable
+    p,
+    span {
```

Comment:
Can you add inputs too? Right now you can't highlight text that you're currently entering.